packages/frontend/web/components/Outbrain.tsx

```diff
@@ -21,9 +56,10 @@ export const OutbrainContainer: React.FC<{
     if (!shouldDisplayOutbrain(config)) {
         return null;
     }
+    const outbrainSelectors = getOutbrainSelectors();
     return (
         <Container>
-            <OutbrainWidget />
+            <OutbrainWidget selectors={outbrainSelectors} />
```

Comment:
maybe this doesn't need to be passed in and can just be generated from the outbrainwidget

Reply:
Done.